shared/scripts/ci-tests.sh

```diff
@@ -66,7 +66,8 @@ setup() {
 }
 
 run_tests() {
-  azk nvm gulp test --slow
+  unset TEST_GREP
```

Comment:
If `TEST_GREP` isn't empty, the test suite can be "grepped" into few tests, not the whole suite.
Since the intention of this script is to run the whole test suite, this cleaning is assuring that all the tests will be run.